tests/test_manifest.py

```diff
@@ -531,3 +533,40 @@ def test_invalid_if_statement(self):
     def test_temporary_must_with_reason(self):
         with pytest.raises(InvalidIfClause, match='"reason" must be set when "temporary: true"'):
             IfClause(stmt='IDF_TARGET == "esp32"', temporary=True)
+
+
+def test_consecutive_or_and_stml_manifest(tmpdir, caplog):
```

Comment:
please rebase and use `tmp_path`

btw, don't have to chdir to `tmp_path` now. it's done in https://github.com/espressif/idf-build-apps/blob/8e421b59e80944d31ce43903dc6362a093558ce5/tests/conftest.py#L22

Reply:
I directly used `IfClause`